webapp/spdx.py

```diff
@@ -0,0 +1,434 @@
+from __future__ import unicode_literals
+
+from license_expression import (ExpressionError, LicenseSymbol, Licensing,
+                                ParseError)
+from webapp.exceptions import ValidationError
+
+# imported from SPDX license list:
+# https://github.com/spdx/license-list
+LICENSES = [
```

Comment:
Rather than keeping the list here in `spdx.py`, would it be possible to read it from a file called e.g. `licenses.yaml` from inside the codebase somewhere?

This would make it easier to talk about the prospect of, e.g., generating that file at build time from some other source, and would I believe be more straightforward to read anyway.